src/EnergyPlus/WaterCoils.cc

```diff
@@ -993,6 +996,13 @@ namespace WaterCoils {
 			PlantLoopScanFlag( CoilNum ) = false;
 		}
 		if ( ! SysSizingCalc && MySizeFlag( CoilNum ) ) {
+
+			// get water coil controller index
+			errFlag = false;
+			GetControllerIndex( WaterCoil( CoilNum ).WaterInletNodeNum, WaterCoil( CoilNum ).ControllerIndex, errFlag );
+			GetControllerName( WaterCoil( CoilNum ).ControllerIndex, WaterCoil( CoilNum ).ControllerName, errFlag );
+
```

Comment:
I did this so I could call ManageControllers from where the coils are simulated. Don't need this anymore.